utils/generator/requirements.txt

```diff
@@ -0,0 +1,5 @@
+xmlschema
```

Comment:
Bilal, docs.google.com link is not working for me, could you please check it?
I committed section for generator and it's dependencies in third_party.md

Reply:
You are right. Sorry about that. This is the working link:
https://docs.google.com/forms/d/1VNR8EUd5b46cQ7uNbCq1fJmnu0askNpUp5dudLKRGpU/viewform

Reply:
We using pylint python library for python code checks/linting. It licensed under GPL license per project page https://github.com/PyCQA/pylint. Could you please confirm we need to remove this dependency and consider other library for code checks (which distributed under supported license)?
All other dependencies is under allowed list of licenses (MIT, BSD3 and Apache2.0)

Reply:
@bilal-alsharifi meanwhile we have used PyLint based on the case that it is already used in [smartdevicelink/sdl_core interface_generator](https://github.com/smartdevicelink/sdl_core/blob/master/tools/InterfaceGenerator/requirements.txt) Is that wrong and outdated usage with the unsupported license too?

Reply:
@theresalech do you know what we should do in that case?

Reply:
By the way, I already tried to replace pylint with flake8 which is licensed by MIT license https://pypi.org/project/flake8/ and it works fine for code checks, so we can replace it without a problem. Both pylint and flake8 are 2 most popular python code check tools so I don't think it will be a problem. flake8 also was among dependencies for sdl_core interface generator.

Reply:
That is great then. Thanks for doing that.

Reply:
No problem, moved this to PR branch